contracts/loot/loot/stats/item.cairo

```diff
@@ -135,6 +137,234 @@ namespace ItemStats {
         dw ItemSlot.HeavyGloves;
     }
 
+    func loot_banned_name_prefixes{syscall_ptr: felt*, range_check_ptr}(index: felt) -> (
+        is_banned: felt
+    ) {
+        let (label_location) = get_label_location(labels);
+        return ([label_location + index - 1],);
+
+        labels:
+        dw TRUE;
```

Comment:
The `labels` values seem to be always a triplet of `(TRUE, FALSE, FALSE)` so an "easier on the eyes" way is to do `index % 3` and check the result. If it's 0, `TRUE, else `FALSE`. Same applies to the function below.

Reply:
thanks for the review, i've refactored the code 👍